demisto_sdk/commands/upload/uploader.py

```diff
@@ -250,13 +264,54 @@ def entity_dir_uploader(self, path: str) -> int:
         return status_code
 
     def pack_uploader(self, path: str) -> int:
-        status_code = 0
+        status_code = SUCCESS_RETURN_CODE
         sorted_directories = sort_directories_based_on_dependencies(get_child_directories(path))
         for entity_folder in sorted_directories:
             if os.path.basename(entity_folder.rstrip('/')) in CONTENT_ENTITIES_DIRS:
                 status_code = self.entity_dir_uploader(entity_folder) or status_code
         return status_code
 
+    def zipped_pack_uploader(self, path: str) -> int:
+
+        zipped_pack = Pack(path)
+
+        try:
+            logger = logging.getLogger('demisto-sdk')
+
+            if not self.pack_names:
+                self.pack_names = [zipped_pack.path.stem]
+
+            if self.notify_user_should_override_packs():
+                zipped_pack.upload(logger, self.client)
+                self.successfully_uploaded_files.extend([(pack_name, FileType.PACK.value) for pack_name in self.pack_names])
+                return SUCCESS_RETURN_CODE
+
+            return ABORTED_RETURN_CODE
+
+        except (Exception, KeyboardInterrupt) as err:
+            file_name = zipped_pack.path.name  # type: ignore
+            message = parse_error_response(err, FileType.PACK.value, file_name, self.log_verbose)
+            self.failed_uploaded_files.append((file_name, FileType.PACK.value, message))
+            return ERROR_RETURN_CODE
+
+    def notify_user_should_override_packs(self):
+        """Notify the user about possible overridden packs"""
+
+        response = self.client.generic_request('/contentpacks/metadata/installed', "GET")
+        installed_packs = eval(response[0])
+        if installed_packs:
+            installed_packs = {pack['name'] for pack in installed_packs}
+            common_packs = installed_packs & set(self.pack_names)  # type: ignore
+            if common_packs:
+                pack_names = '\n'.join(common_packs)
+                click.secho(f'This command will overwrite the following packs:\n{pack_names}.\n'
+                            'Any changes made on XSOAR will be lost.\n'
+                            'Are you sure you want to continue? Y/N', fg='bright_red')
```

Comment:
```suggestion
                            'Are you sure you want to continue? Y/[N]', fg='bright_red')
```
That's for indicating what is the default value.